src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -509,7 +509,7 @@ public String getAudioCodec() {
 	@Override
 	public String toString() {
 		StringBuilder result = new StringBuilder();
-		if (!getLang().equals("und")) {
+		if (!getLang().equals("und") && getLang() != null) {
```

Comment:
@Sami32 You have to move the null check first, or you will still get NPE:

```java
if (getLang() != null && !getLang().equals("und")) {
```
They are checked in that order...